public/locales/en/panel-time.json

```
@@ -0,0 +1,68 @@
+{
+  "select-time-title": "Select Time",
+  "error": {
+    "outside-range": "Can't interact with dates outside the range April 20, 271821 BC to Sep 13, 275760 AD."
+  },
+  "simulation-speed-title": "Simulation Speed",
+  "realtime": "Realtime",
+  "now": "Now",
+  "time-parts": {
+    "Milliseconds_one": "Milliseconds",
+    "Milliseconds_other": "Milliseconds",
+    "Seconds_one": "Second",
+    "Seconds_other": "Seconds",
+    "Minutes_one": "Minute",
+    "Minutes_other": "Minutes",
+    "Hours_one": "Hour",
+    "Hours_other": "Hours",
+    "Days_one": "Day",
+    "Days_other": "Days",
+    "Weeks_one": "Week",
+    "Weeks_other": "Weeks",
+    "Months_one": "Month",
+    "Months_other": "Months",
+    "Years_one": "Year",
+    "Years_other": "Years"
+  },
+  "simulation-increment": {
+    "select-unit-label": "Display Unit"
+  },
+  "quick-adjust": {
+    "title": "Quick Adjust"
+  },
+  "delta-time-step-control": {
+    "aria-label": {
```

In another PR you used the plural `aria-labels` when grouping multiple labels like this. I preferred that - please do it here too
```suggestion
    "aria-labels": {
```